variables.tf

```diff
@@ -278,6 +278,12 @@ variable "monitoring_sns_protocol" {
   default     = "email"
 }
 
+variable "monitoring_enable_detailed_monitoring" {
+  description = "Enable or disable detailed monitoring"
```

Comment:
Could probably add a bit more explanation for this variable here or as an example in the readme.